mailbox/lucene/src/main/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndex.java

```diff
@@ -771,70 +760,42 @@ private static boolean hasAttachment(MailboxMessage membership) {
     }
 
     private String toSentDateField(DateResolution res) {
-        switch (res) {
-            case Year:
-                return SENT_DATE_FIELD_YEAR_RESOLUTION;
-            case Month:
-                return SENT_DATE_FIELD_MONTH_RESOLUTION;
-            case Day:
-                return SENT_DATE_FIELD_DAY_RESOLUTION;
-            case Hour:
-                return SENT_DATE_FIELD_HOUR_RESOLUTION;
-            case Minute:
-                return SENT_DATE_FIELD_MINUTE_RESOLUTION;
-            case Second:
-                return SENT_DATE_FIELD_SECOND_RESOLUTION;
-            default:
-                return SENT_DATE_FIELD_MILLISECOND_RESOLUTION;
-        }
-    }
-
-    private static Calendar getGMT() {
-        return Calendar.getInstance(TimeZone.getTimeZone("GMT"), Locale.US);
+        return switch (res) {
+            case Year -> SENT_DATE_FIELD_YEAR_RESOLUTION;
+            case Month -> SENT_DATE_FIELD_MONTH_RESOLUTION;
+            case Day -> SENT_DATE_FIELD_DAY_RESOLUTION;
+            case Hour -> SENT_DATE_FIELD_HOUR_RESOLUTION;
+            case Minute -> SENT_DATE_FIELD_MINUTE_RESOLUTION;
+            case Second -> SENT_DATE_FIELD_SECOND_RESOLUTION;
```

Comment:
Where is the default case handled? Are we exhaustive?

Reply:
`DateResolution` is the enum
then default is not need

Reply:
My mean is 
```
    public enum DateResolution {
        Second, Minute, Hour, Day, Month, Year
    }
```
all enum is handled here, so the `default` is unnecessary

IDE detected:
  
![Screenshot_20241001_094802](https://github.com/user-attachments/assets/ac0e1ac7-8552-411d-9129-7d866f291fee)